test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1474,9 +1487,15 @@ def tearDown(self):
 
         for sched in self.scheds:
             self.scheds[sched].cleanup_files()
+        self.server.del_hooks_nodes_qs()
+        self.server.load_configuration(self.saved_file)
+        self.du.rm(path=self.saved_file)
```

Comment:
Do you really need to remove the saved configuration? Why save and delete for every test? You only need to save it once in setUp() and then just load it in tearDown() right?

Reply:
Hi Ravi,

As per the EDD : https://pbspro.atlassian.net/wiki/spaces/PD/pages/557088789/Design+for+a+supported+way+to+change+default+setup+in+PTL
- We have to bring back PBS to the same custom setup on which it started.
- Saved configuration in setup() has to be loaded in teardown() everytime.
- When --use-current-setup option is used, tests should run on given custom setup.

Instead of saving the configurations for each test case. I could save configuration the first time when setup() is called for the first time in a test suite

Reply:
right, that's exactly what i was suggesting. Save it only once, don't save again if already saved, then in tearDown() you don't need to delete self.saved_file

Reply:
updated changes by adding class variable: PBSTestSuite.config_saved as mentioned in design doc.